src/components/user/UserResetPassword.component.vue

```diff
@@ -0,0 +1,148 @@
+<template>
```

Comment:
Komentarz ogólny. Czy componenty potrzebują suffix .component skoro są w lokalizacji src/components? Tasiemiec trochę. Trzymaj się konwencji jeśli używasz UserResetPassword to nie używaj PasswordRecovery i na odwrót.

Reply:
Pomyśl czy component da się rozbić na mniejsze atomowe części. Znaczniki się ciężko czyta.

Reply:
Zrobię to, gdy zmerguje twój refactor, narazie leci do backlogu.